orcoursetrion/config.py

```diff
@@ -24,6 +24,12 @@
     # GitHub API URL
     ('ORC_GH_API_URL', 'https://github.mit.edu/api/v3/'),
 
+    # GitHub Committer Name
+    ('ORC_GH_NAME', 'Orcoursetrion ODL'),
+
+    # GitHub Committer Email
+    ('ORC_GH_EMAIL', 'orcoursetrion@example.com'),
```

Comment:
My only reasoning is "explicit over implicit", since the "keys" here aren't enforced as a set, and the values wouldn't be predictable for duplicates - and so this structure is trying to be a dict. :wink: I see that probably doesn't matter so much here, though, since these are just static defaults and there aren't even a lot of them.

I'm cool with either leaving it as is or refactoring.